test/extended/util/cli.go

```diff
@@ -66,6 +82,13 @@ func (c *CLI) Username() string {
 	return c.username
 }
 
+// AsAdmin changes current config file path to the admin config.
+func (c *CLI) AsAdmin() *CLI {
```

Comment:
+1 @rhcarvalho.  AsAdmin is fine.  AsAnAdmin is also fine, though it looks worse.